mackerel-plugin-mysql/lib/mysql.go

```diff
@@ -700,7 +718,7 @@ func parseInnodbStatus(str string, p *map[string]float64) {
 			(*p)["pending_normal_aio_writes"], _ = atof(record[7])
 			continue
 		}
-		if strings.Index(line, "ibuf aio reads") == 0 {
+		if strings.Index(line, "ibuf aio reads") == 0 && len(record) >= 10 {
```

Comment:
[question]
Is this just for safe? Or are there real cases what `len(record) < 10`?

Reply:
because it failed to test in this line:
https://github.com/mackerelio/mackerel-agent-plugins/blob/2ba382fdfe71ec75b144434c8e2379a9ece9ae2f/mackerel-plugin-mysql/lib/mysql_test.go#L817
BTW, I have never seen this result in real environment.